AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -340,9 +340,10 @@ nero.com###fancybox-wrap
 nero.com###fancybox-overlay
 ! https://github.com/AdguardTeam/AdguardFilters/issues/44364
 freiepresse.de###aboBanner
-! https://github.com/AdguardTeam/AdguardFilters/issues/44090
+! https://github.com/AdguardTeam/AdguardFilters/issues/80042
 flipkart.com##div[data-id][style] > div[data-tkid^="en_"]
 flipkart.com#?#div[style] > div[class*="col-"] > div[class] > div[data-id][style]:has(> div[data-tkid^="en_"])
+flipkart.com#?#div[class^="css-"] > div:has(> div[dir]:contains(/^AD$/))
```

Comment:
As i remember correctly, there was a case of such elements and we decided not to block them, as they are paid by users and don't show up somewhere else again after blocking. I might be wrong, @Sergey-Lyapin, check this please.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/114696027-e584c800-9d24-11eb-86f8-9e46ba4ab718.png)</details><br/>

Reply:
What do you mean by "as they are paid by users"?

Reply:
I see these elements are blocked as self-promo by Blaz

```
! https://github.com/AdguardTeam/AdguardFilters/issues/44090
flipkart.com##div[data-id][style] > div[data-tkid^="en_"]
flipkart.com#?#div[style] > div[class*="col-"] > div[class] > div[data-id][style]:has(> div[data-tkid^="en_"])
```

Reply:
Yes, the rules I have crafted are for mobile.

Reply:
what about something like this?
```
flipkart.com#?##_parentCtr_ > div[style^="transform: translate"][style*="width: 360px;"]:has(div[dir="auto"]:contains(/^AD$/))
```

Reply:
Mine is already narrowed enough.

Reply:
And this rule doesn't work on my end.